package-lock.json

```diff
@@ -1,13 +1,16 @@
 {
   "name": "autometrics",
   "version": "0.0.1",
-  "lockfileVersion": 3,
+  "lockfileVersion": 2,
```

Comment:
Are you using an older NPM version that caused the lock file version to be lowered?